ambari-logsearch/ambari-logsearch-server/src/main/java/org/apache/ambari/logsearch/configurer/LogSearchConfigConfigurer.java

```diff
@@ -57,9 +61,13 @@ public void run() {
         logger.info("Started thread to set up log search config");
         while (true) {
```

Comment:
Should the number of retries limited?

Reply:
Not necessary, because the Logsearch API indicates the failure if the ZK is not available.